design-tokens/assets/android/StyleDictionary.kt

```diff
@@ -0,0 +1,19 @@
+
+
+// Do not edit directly
+// Generated on Fri, 20 Jan 2023 15:09:34 GMT
+
+
+
+package io.element.compound.tokens;
+
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.*
+
+object StyleDictionary {
+  /** The main brand color */
+  val brand = #0dbd8b
+  /** The default font size */
+  val sizeDefault = 256.00.dp
```

Comment:
This is OK, hence the decimal are not required (so it could be `256.dp`). As a side note, 256 dp is huge, default font size is generally 16 dp. I see that in the source file this is `16px`. So it is OK to convert to `16.dp`.